src/ansys/aedt/core/visualization/post/fields_calculator.py

```diff
@@ -581,6 +625,179 @@ def calculator_write(self, expression, output_file, setup=None, intrinsics=None)
         self.ofieldsreporter.CalcStack("clear")
         return True
 
+    @pyaedt_function_handler()
+    def export(
+        self,
+        quantity,
+        solution=None,
+        variations=None,
+        output_file=None,
+        intrinsics=None,
+        phase=None,
+        sample_points=None,
+        export_with_sample_points=True,
+        reference_coordinate_system="Global",
+        export_in_si_system=True,
+        export_field_in_reference=True,
+        grid_type=None,
+        grid_center=None,
+        grid_start=None,
+        grid_stop=None,
+        grid_step=None,
+        is_vector=False,
+        assignment="AllObjects",
+        objects_type="Vol",
+    ):
+        """Export the field quantity in the top register to a file, mapping it to a grid of points.
```

Comment:
```suggestion
        """Export the field quantity of the stack register to a file, mapping it to a grid of points.
```
Not sure about this change though.